src/web/hooks/usePageFilter.js

```diff
@@ -47,12 +47,14 @@ const useDefaultFilter = pageName =>
  *
  * @param {String} pageName Name of the page
  * @param {Object} options Options object
+ * @param {String} gmpName Name of the GMP for filtering
```

Comment:
I think GMP name may be more clear
```suggestion
 * @param {String} gmpName GMP name for filtering
```